itest/litd_custom_channels_test.go

```diff
@@ -2745,6 +2745,160 @@ func testCustomChannelsLiquidityEdgeCasesGroup(ctx context.Context,
 	testCustomChannelsLiquidtyEdgeCasesCore(ctx, net, t, true)
 }
 
+// testCustomChannelsMultiRFQReceive tests that a node creating an invoice with
+// multiple RFQ quotes can actually guide the payer into using multiple private
+// taproot asset channels to pay the invoice.
+func testCustomChannelsMultiRFQReceive(ctx context.Context, net *NetworkHarness,
+	t *harnessTest) {
+
+	lndArgs := slices.Clone(lndArgsTemplate)
+	litdArgs := slices.Clone(litdArgsTemplate)
+
+	// Explicitly set the proof courier as Zane (now has no other role
+	// other than proof shuffling), otherwise a hashmail courier will be
+	// used. For the funding transaction, we're just posting it and don't
+	// expect a true receiver.
+	zane, err := net.NewNode(
```

Comment:
A recent refactor removed all the Zane nodes that weren't involved in the test other than serving as the universe. Using Charlie here would save around 2 seconds of test preparation time, see other tests (we just need to derive the listening port for Charlie upfront).